manimlib/animation/transform.py

```diff
@@ -152,7 +152,7 @@ def __init__(self, method, *args, **kwargs):
         method is a method of Mobject, *args are arguments for
         that method.  Key word arguments should be passed in
         as the last arg, as a dict, since **kwargs is for
-        configuration of the transform itslef
+        configuration of the transform itself
```

Comment:
this is better suited in a general bug fix PR (so we have a proper organization)